python-sdk/rebuff/rebuff_pythonsdk.py

```diff
@@ -0,0 +1,237 @@
+import secrets
+
+from typing import Any, Optional, Tuple
+from detect_pi_heuristics import detect_prompt_injection_using_heuristic_on_input
+from detect_pi_vectorbase import init_pinecone, detect_pi_using_vector_database
+from detect_pi_openai import render_prompt_for_pi_detection, call_openai_to_detect_pi
+import requests
+from pydantic import BaseModel
+
+
+class Rebuff_Detection_Response(BaseModel):
```

Comment:
Since this code isn't parsing user input or doing data validation, I don't see any benefit to inheriting from `BaseModel`. Is there another reason why there's benefit to inheriting from that?

Reply:
I think there was some discussion for keeping one (likely Python) schema and generating ts-schema from it where `BaseModel` might help. Though still a work in progress.